packages/dataviews/src/stories/dataviews.story.tsx

```diff
@@ -115,13 +121,15 @@ export const Default = ( {
 			isItemClickable={ () => hasClickableItems }
 			defaultLayouts={ defaultLayouts }
 			config={ { perPageSizes } }
+			padding={ padding }
```

Comment:
Why do we need this property? Perhaps I've misinterpreted the conversation in this PR and this [other one](https://github.com/WordPress/gutenberg/pull/73390#discussion_r2549115817), so correct me if I'm wrong: aren't we promoting the use of CSS Custom Properties to style DataViews? If so, why having both things?

cc @jameskoster

Reply:
In other words, how consumers are expected to tweak the padding (or background in the other PR):

```css
:root {
  --wp-dataviews-padding-block-start: /* ... */
}
```

or 

```js
<DataViews
  padding={ ... }
/> 
```

I thought Jay argued for the 1st approach, given that's how other components in the upcoming new design system (`@wordpress/theme` and `@wordpress/ui`) and the existing `@wordpress/components` work.